conformance.adoc

```diff
@@ -568,9 +568,6 @@ variable, then those coordinate variables which correspond to node coordinate va
  **`bounds`** attribute that names the corresponding node coordinate.
```

Comment:
Following the conclusion of the "nodes" discussion, this check needs updating. How about:

* If a coordinate variable named by a **`coordinates`** attribute carried by the geometry container variable or its parent data variable has a **`nodes`** attribute, then the **`nodes`** attribute must be a string whose value is a single variable name. The specified variable must be a node coordinate variable that exists in the file.